app/models/enqueuer.rb

```diff
@@ -0,0 +1,49 @@
+class Enqueuer
+  DELAY_THRESHOLD = 30
+  def enqueue_faxes
+    SnapApplication.
+      faxable.
+      untouched_since(DELAY_THRESHOLD.minutes.ago).
+      find_each do |snap_application|
+        create_and_enqueue_export(destination: :fax,
+                                  snap_application: snap_application)
+      end
+  end
+
+  def create_and_enqueue_export(params)
+    enqueue(Export.create(params))
+  end
+
+  def self.create_and_enqueue_export(params)
+    new.create_and_enqueue_export(params)
+  end
+
+  def create_and_enqueue_export!(params)
```

Comment:
this `Enqueuer` class does appear to be doing more than enqueueing -- its also creating an export! did you think about using a different name or dependency injection to do the creation outside of the class?

Reply:
I don't see creation of the export that needs to be enqueued as an independent piece of responsibility from enqueueing; but I am happy to hear name suggestions that you feel are more reasonable for this extraction?

Reply:
Someone very Ruby smart once told me that whenever there is an `and` in a class or method name, that means that two things are happening. Not terrible to have two things here (much better than 3+!) was just wondering if it would make sense to pass in the export, so it would be something like

```
Enqueuer.enqueue_export(
  destination: :fax,
  snap_application: current_snap_application,
  export: Export.create!(params),
)
```

Reply:
I think what you're smelling here is totally valid; but I would rather we came up with a solution that maintains the current level of encapsulation as opposed to one that moves responsibility up.

So yes, naming is hard. However, my impression is that we're trying to avoid over-investing at this time; and I'm feeling like this is a six in one half dozen in another given that the results would shove a bunch of responsibility into the controller for making sure that jobs can be audited and tracked correctly.